### ISSUE TITLE
Support for .NET 8

### ISSUE THREAD

**Issue by bangbaew** [dotnet-outdated/dotnet-outdated#482]
I'm using .NET 8 and when I try to run this tool it says
```sh
You must install or update .NET to run this application.

App: /root/.dotnet/tools/dotnet-outdated
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '7.0.0' (x64)
.NET location: /usr/share/dotnet

The following frameworks were found:
  8.0.0-preview.7.23375.6 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=7.0.0&arch=x64&rid=linux-x64&os=debian.12
```
Please add support for .NET 8 because it's going to be an LTS one while .NET 7 is only Standard Term Support.

**Comment by Deadvi5** [comment]
Any update on this? I'm also interested in support for .NET 8

**Comment by gerardog** [comment]
Have anyone actually tested the tool on .NET 8?

The pasted error message indicates that `dotnet-outdated` can't run because it requires the .NET 7 runtime, which is not installed. You can install it by following the link at the end of the error message.

If you see this error but mentioning `'Microsoft.NETCore.App', version '3.0.0'` instead, then you have installed the deprecated version of the tool and need to update it by running

```
dotnet tool uninstall --global dotnet-outdated
dotnet tool install --global dotnet-outdated-tool
```

**Comment by kyurkchyan** [comment]
I had a similar issue. Today, .NET 8 was made generally available. After I updated everything and ran `dotnet workload restore` and `dotnet workload update` everything started working for me.

**Comment by jeroenhabets** [comment]
@slang25 , @reduckted , @martincostello , @gigi81 please urgently add .Net 8 support, it's now GA after all so more people  will hit this.

After upgrading to the latest version it stopped working with same error as in the description, the `dotnet workload restore` and `dotnet workload update` workaround does not work for me.

Changing `<TargetFrameworks>net6.0;net7.0</TargetFrameworks>` into `<TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>` might do the trick...

**Comment by slang25** [comment]
I'm on it 👍

**Comment by leo-costa** [comment]
> I'm on it 👍
 :) 

https://github.com/dotnet-outdated/dotnet-outdated/pull/500

**Comment by slang25** [comment]
I've just published a [4.6.0](https://github.com/dotnet-outdated/dotnet-outdated/releases/tag/v4.6.0), it will take a few minutes to [show up on nuget](https://www.nuget.org/packages/dotnet-outdated-tool/4.6.0).

**Comment by solrevdev** [comment]
> I've just published a [4.6.0](https://github.com/dotnet-outdated/dotnet-outdated/releases/tag/v4.6.0), it will take a few minutes to [show up on nuget](https://www.nuget.org/packages/dotnet-outdated-tool/4.6.0).

Thank you. works great. much appreciated 🙏🏼️

**Comment by nelsonmorais** [comment]
Anyone facing the issue mentioned on #514 while using version 4.6.0? Any suggestions?
